sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -213,7 +213,7 @@ case class Multiply(left: Expression, right: Expression)
 case class Divide(left: Expression, right: Expression)
     extends BinaryArithmetic with NullIntolerant {
 
-  override def inputType: AbstractDataType = NumericType
```

Comment:
Looks like div is from mysql (see http://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html).

```
DIV	Integer division
/	Division operator
```